src/components/button/theme.ts

```diff
@@ -0,0 +1,56 @@
+import { SystemStyleObject } from '@chakra-ui/theme-tools';
+import { ComponentStyleConfig } from '@chakra-ui/theme';
+
+import { sizes as buttonHeights, typography } from '../../theme/shared';
+
+const baseStyle: SystemStyleObject = {
+  borderRadius: 'sm',
+  fontWeight: 'bold',
+};
+
+const sizes = {
+  md: {
+    height: buttonHeights.md,
+    px: 'md',
+    ...typography.heading5,
+  },
+  lg: {
+    height: buttonHeights.lg,
+    px: 'lg',
+    ...typography.heading4,
+  },
+  xl: {
+    height: buttonHeights.xl,
+    px: 'xl',
+    ...typography.heading4,
```

Comment:
For the `px` and `typography` I will check with the Designer if we can spiffily it. I will let you know as soon as I have an aswer. Thanks!

Reply:
That's cool!

Reply:
Btw, @GutierrezGo , there's a button can go over 2 lines.

<img width="490" alt="image" src="https://user-images.githubusercontent.com/3179244/171819813-25331f8f-c64b-449e-a6a2-38a719778ae5.png">

As I can see, UX decided to use the fixed height approach. How can we adapt with this multiple-line button in this case?

Reply:
I have checked with Sergiu and the text on the button should be one line as much as possible. The designers will always check that the text fit one line in ALL the languages.
In case that the text takes two lines the width will increase. If the text is too long, we will give a try truncating the text (this point is still in review). This is a starting point for the button component, we will face many cases where we will need to adapt the component again. But for now, we will roll out with the fix height and make the width dynamic as much as possible.
Is it make sense for you?

Reply:
I have checked with Sergiu and regarding the `typography` with the fontSize, he needs to doble check if we can have `fontSize: 16px` in the `md button`. 

But what it's sure for now is that we can give the same `px` for all the button variations with `16px`.

Reply:
I've updated the `px` to the same value `lg` = `16px` for every size of button

Reply:
About the `fontSize` we need to keep the `14px` on the `md` button. The designers have double checked that case and 14px would be preferred because of the size of the button. Thank you!